spec/controllers/page_controller_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe Account::PagesController, type: :controller do
+  context 'Page' do
+    let!(:page) { FactoryBot.create(:page) }
+
+    it 'Get#index' do
```

Comment:
stick to the common conventions

Reply:
something like "GET #INDEX" ... just check other specs